src/mito2/src/read/scan_region.rs

```diff
@@ -104,6 +96,17 @@ impl Scanner {
             Scanner::Unordered(unordered_scan) => unordered_scan.input().file_ids(),
         }
     }
+
+    /// Sets the target partitions for the scanner. It can controls the parallelism of the scanner.
+    pub(crate) fn set_target_partitions(&mut self, target_partitions: usize) {
+        use store_api::region_engine::{PrepareRequest, RegionScanner};
```

Comment:
Move the `use` out of the function.

Reply:
This method is under `#[cfg(test)]` so moving the `use` to outside will cause `unused` warning.